### PR TITLE
Update audio - removing deep_buffer mode

### PR DESCRIPTION
**Pull request by ernesst**
- Adjust the earpiece volume to a bit lower, 
- Remove quirk and deep_buffer mode,
- Select the correct Id for the voice speaker,
- Remove a audio configuration file not in used. 
- voice speaker working

Thanks to https://github.com/fredldotme/device-sony-suzu/commit/57bc95deb28656d7f508d0e4f8718fd6dd18b2a8

@Vince1171

### CODE REVIEW

File: audio/mixer_paths_tasha.xml [hunk 1]
```diff
@@ -2182,6 +2182,7 @@
         <ctl name="SLIM_0_RX Channels" value="One" />
         <ctl name="RX INT0_1 MIX1 INP0" value="RX0" />
         <ctl name="RX INT0 DEM MUX" value="CLSH_DSM_OUT" />
+	<ctl name="RX0 Digital Volume" value="70" />
```
Comment (Vince1171) [inline audio/mixer_paths_tasha.xml]: can you use spaces instead of a tab plz? :) @ernesst

**Review by Vince1171 — APPROVED** [review]

**Comment by Flohack74** [comment]
The thing is that deep buffer mode has an impact on the CPU processing of audio samples. We should make sure that this does not force music playback into CPU-intense operations, as this will drain battery

**Comment by ernesst** [comment]
@Flohack74 thanks for the comment, i did notice a higher battery consumption in the battery but nothing crazy. I'm still impress of having a phone running a day and half.
I've checked the music playback it's around 30% consumption during use.

**Comment by Flohack74** [comment]
Ok so you are saying that music-app loads CPU to 30% ?

**Comment by ernesst** [comment]
The exact top while playing music 
image

**Comment by Flohack74** [comment]
Ouch that hurts ;) - Yeah, but the percentage does not add up, strange... Hmm anyways, we know smth for the future to be improved...

**Pull request merged**